src/transformers/keras_callbacks.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from tensorflow.keras.callbacks import Callback
+
+
+class PushToHubCallback(Callback):
+    def __init__(
+        self,
+        save_strategy="epoch",
+        save_steps: Optional[int] = None,
+        repo_path_or_name: Optional[str] = None,
+        repo_url: Optional[str] = None,
```

Comment:
I would prefer we use the same arguments as the `Trainer`: `output_dir`, `hub_model_id` and `hub_token` to be consistent between TF and PyTorch.